redaxo/src/core/lib/be/controller.php

```diff
@@ -199,6 +200,19 @@ public static function appendLoggedInPages()
                 ->setHasLayout(false)
                 ->setPath(rex_path::core('pages/system.phpinfo.php')),
             );
+
+        // ---------- Phpmailer
+        self::$pages['phpmailer'] = (new rex_be_page_main('addons', 'phpmailer', rex_i18n::msg('phpmailer_title')))
```

Comment:
Ist die Frage, ob die Page wirklich noch in den addons-Block gehört. Jan hat in seinen PRs die glaube ich nach oben gepackt in den Main-Block.